src/main/java/io/temporal/samples/helloworld/HelloWorldStarter.java

```diff
@@ -0,0 +1,31 @@
+package io.temporal.samples.helloworld;
+// @@@SNIPSTART java-hello-world-sample-workflow-starter
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class HelloWorldStarter {
+
+  public static void main(String args[]) throws Exception {
+
+    // gRPC stubs wrapper that talks to the local docker instance of temporal service.
+    WorkflowServiceStubs service = WorkflowServiceStubs.newInstance();
+    // client that can be used to start and signal workflows
```

Comment:
This comment is not really complete. Why only start and signal? We can start, signal, query, cancel, terminate, etc.